proto/proto/store/rpc.proto

```diff
@@ -201,6 +203,14 @@ message CheckNullifiersByPrefixResponse {
     }
     // List of nullifiers matching the prefixes specified in the request.
     repeated NullifierUpdate nullifiers = 1;
+    // The block number of the last check included in this response.
+    //
+    // For chunked responses, this may be less than request.block_to.
+    // If it is less than request.block_to, the user is expected to make a subsequent request
+    // starting from the next block to this one (ie, request.block_from = block_num + 1).
+    fixed32 block_num = 2;
+    // Current chain tip
+    fixed32 chain_tip = 3;
```

Comment:
Similar to the previous comment, I'd change the order of these to be:
```proto
message CheckNullifiersByPrefixResponse {
    ...
    fixed32 chain_tip = 1;
    fixed32 block_num = 2;
    repeated NullifierUpdate nullifiers = 3;
}
```
And if we could update `SyncStorageMapsResponse` and `SyncAccountVaultResponse` to follow the same convention, that would be great.